src/style/variables.scss

```diff
@@ -1,4 +1,25 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
+
 $iui-border-radius: 3px;
 $iui-blur-filter: blur(5px);
+
+
+// All inputs
```

Comment:
What are these? 😀

Reply:
Yeah, these looks like really specific ones to our components.. I would prefer to keep global variables only those useful for external users too (more generic)...

These inputs ones are again huge and with some calculations, which are hard to figure out by just reading :(

Reply:
In description @FlyersPh9 said:

> Removed /inputs/variables.scss and instead placed those variables in /global/variables.scss because they are used by more than just text field & area.

🤔 Are all of these used by more than just inputs?

Reply:
Some of these variables are used by:
- Button
- Fieldset
- Text field
- Text area
- Select
- Menu
- Tag

Reply:
Sounds bad... What Button has to do with input? I know we want to align them, but.. I would think then to rename those variables to something more descriptive...

Reply:
Technically [a button is a type of input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) but I am open to changing the variable names.

Reply:
See #118 😄

Reply:
Lets change variable to something more explanatory and we can export them all, I guess...

Reply:
Thoughts on leaving the variables as `$iui-input-*` and then getting rid of [the `/inputs/` folder](https://github.com/iTwin/iTwinUI/tree/main/src/inputs) so there is less confusion about what is / is not an input?  [`input.scss`](https://github.com/iTwin/iTwinUI/blob/main/src/inputs/input.scss) would need to be renamed to `text-field.scss` as well to help eliminate any confusion.

Reply:
I do not think that renaming our files, but keeping `iui-input-*` variables name would change anything. YOu can ask anyone, but if they see input, it is input :D
I mean.. couldnt we make that name something more generic or for the purpose it is used? I am not sure the prupose, but we could think of.. `interactive-element-height` or something...